src/test/common/page/page.js

```diff
@@ -73,6 +73,15 @@ class Page {
     }
   }
 
+  async findByXPath (xpath) {
+    try {
```

Comment:
All the other find by methods wait for the element first before finding it. I'm guessing this isn't need?

Reply:
This isn't needed, I think because where we use it the page is already loaded. I also couldn't figure out how to make that part work with xpath.